workspaces/marketplace/plugins/marketplace-backend/package.json

```diff
@@ -44,7 +45,6 @@
     "@red-hat-developer-hub/backstage-plugin-marketplace-common": "workspace:^",
     "express": "^4.17.1",
     "express-promise-router": "^4.1.0",
-    "yaml": "^2.7.1",
```

Comment:
`plugins/marketplace-backend/src/installation/FileInstallationStorage.ts` is using the yaml package. Please move this back from devDependencies to dependencies.